packages/mobile/src/components/overflow-menu-drawer/CollectionOverflowMenuDrawer.tsx

```diff
@@ -96,6 +96,9 @@ const CollectionOverflowMenuDrawer = ({ render }: Props) => {
     [OverflowAction.VIEW_ARTIST_PAGE]: () => {
       navigation?.push('Profile', { handle })
     },
+    [OverflowAction.EDIT_ALBUM]: () => {
+      navigation?.push('EditPlaylist', { id })
```

Comment:
Might be worth renaming to edit collection?

Reply:
Running out of time today and want to get it merged. I'll consider this tomorrow as a follow-up